exercises/practice/satellite/package.yaml

```diff
@@ -0,0 +1,24 @@
+name: satellite
+version: 1.0.0.0
+
+dependencies:
+  - base
+  - containers
```

Comment:
`containers` isn't needed anymore.

Reply:
You are correct, it is need only in the example solution which uses Data.Set.